src/routes/(inner)/docs/getting-started/+page.svelte

```diff
@@ -55,4 +58,19 @@ npm install @skeletonlabs/floating-ui-svelte
 		/>
 		<CodeBlock lang="html" code={`<div class="floating">Some floating element.</div>`} />
 	</section>
+	<!-- Ceveats -->
+	<section class="space-y-8">
+		<h2 class="h2">Caveats</h2>
+		<h3 class="h3">SSR</h3>
+		<p>
```

Comment:
Go ahead and collapse these two paragraphs into one.

Reply:
Done